.github/workflows/base-image.yml

```diff
@@ -26,7 +29,9 @@ jobs:
         run: echo "${{ secrets.GITHUB_TOKEN }}" | docker login ghcr.io -u $ --password-stdin
       - name: Push tagged image
         if: github.event_name == 'push' && github.ref_type == 'tag'
-        run: docker push ghcr.io/cnpem/lnls-debian-11-epics-7:$TAG
+        run: docker push $REGISTRY/$IMAGE:$TAG
       - name: Push latest image
         if: github.event_name == 'push' && github.ref_type == 'branch'
-        run: docker tag ghcr.io/cnpem/lnls-debian-11-epics-7:$TAG ghcr.io/cnpem/lnls-debian-11-epics-7:latest && docker push ghcr.io/cnpem/lnls-debian-11-epics-7:latest
+        run: |
+          docker tag $REGISTRY/$IMAGE:$TAG $REGISTRY/$IMAGE:latest
+          docker push $REGISTRY/$IMAGE:latest
```

Comment:
This might be going in the opposite direction of what you intended, but I think the correct behavior is actually not publishing anything if we aren't building in the main repository :thinking: 

It'd be a big surprise to fork a repo and suddenly you're filling up your personal registry...

Therefore, I'd consider adding another condition to the if for this block, where the repository has to be in our org.

Reply:
Okay, I haven't considered that it will fill up one's registry even when they only want to contribute upstream.

I'll fix that up. Thanks for the feedback!